src/public/javascripts/services/tree.js

```diff
@@ -635,7 +635,7 @@ async function createNote(node, parentNoteId, target, extraOptions = {}) {
         extraOptions.saveSelection = false;
     }
 
-    if (extraOptions.saveSelection) {
+    if (extraOptions.saveSelection && window.cutToNote) {
```

Comment:
As it is now this does not express the intention well (the fact it is trying to check that CKEditor is initialized). Could you encapsulate it into a function like `isCKEditorInitialized` which would be more explanatory ...

Reply:
Good idea. How's this?